spec/article-model-spec.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
```

Comment:
I generally don't bother testing getters and setters - unless they have side effects (e.g. setting a value results in an event of some sort) they are just extensions of state rather than representing any real logic. We can test them more effectively by testing the code which calls/uses them.